src/components/DropdownForm/DropdownForm.tsx

```diff
@@ -0,0 +1,56 @@
+import React, {
+  useCallback, useEffect, useRef, useState,
+} from 'react';
+import debounce from 'lodash.debounce';
+import { Person } from '../../types/Person';
+
+type Props = {
+  querry: string;
+  onQuerry: (value: string) => void;
+  onAppliedQuerry?: (value: string) => void;
+  selectedPerson: Person | null,
+  onDropdownFocus?: (value: boolean) => void;
+};
+
+export const DropdownForm: React.FC<Props> = ({
+  querry,
+  onQuerry,
+  onAppliedQuerry = () => { },
+  selectedPerson,
+  onDropdownFocus = () => { },
+}) => {
+  const [defoultImputFocus, setDefoultImputFocus] = useState(false);
+  const ApplyQuerry = useCallback(debounce(onAppliedQuerry, 1000), []);
```

Comment:
the constant name should be in camelCase or Upper Upper
```suggestion
  const applyQuerry = useCallback(debounce(onAppliedQuerry, 1000), []);
```